webpack.config.js

```diff
@@ -113,7 +113,8 @@ module.exports = {
     alias: {
       css: path.join(__dirname, "/troposphere/static/css/"),
       images: path.join(__dirname, "/troposphere/static/images/"),
-      highcharts: "highcharts-commonjs"
+      highcharts: "highcharts-commonjs",
+      react: path.join(__dirname, "/node_modules/react")
```

Comment:
Yes it seems like listing react as a peer dependency would suggest that be the case but it still gets installed into toroposphere-ui's node_modules. I deleted the node_modules folder and `npm i` just to make sure.   
```
root@vagrant:/opt/dev/troposphere-ui# ls node_modules/
babel-cli            babel-preset-stage-0  radium            react-dom     react-tap-event-plugin
babel-core           babel-preset-stage-2  randomcolor       react-height  tinycolor2
babel-eslint         eslint                react             react-ink
babel-preset-es2015  eslint-plugin-react   react-collapse    react-motion
babel-preset-react   material-ui           react-dimensions  react-scroll
root@vagrant:/opt/dev/troposphere-ui# 
```
The error I got was the one I think you have seen before that's why I kew what the issue was. 
```
vendor.js:22056 Uncaught Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://fb.me/react-refs-must-have-owner).
```

This solution resolves the issue but perhaps there is a better one.